salt/modules/yumpkg.py

```diff
@@ -54,6 +54,7 @@
 import salt.utils.systemd
 import salt.utils.versions
 from salt.utils.versions import LooseVersion as _LooseVersion
+from salt.utils.environment import get_module_environment
```

Comment:
Same as with apt, this would add `get_module_environment` to the `__salt__` dunder.